decentraland/unity-renderer
# feat: Report scene button on minimap hud

## What does this PR change?

Adds the option to report a scene using the minimap menu.

## How to test the changes?

1. Open the 3dots menu of the scene in the minimap. It will show a submenu with the report button
2. Clicking on it, will open a Google report form.
3. Scene location should be prepopulated

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
